spring-cloud-gcp-starters/spring-cloud-gcp-starter-storage/pom.xml

```diff
@@ -26,5 +26,10 @@
 			<groupId>org.springframework.cloud</groupId>
 			<artifactId>spring-cloud-gcp-storage</artifactId>
 		</dependency>
```

Comment:
So the Servlet application context was failing then? Does it have a different mechanism to load resources?